go/interpreter/lfvm/instructions.go

```diff
@@ -668,8 +670,8 @@ func opSelfdestruct(c *context) (status, error) {
 	return statusSelfDestructed, nil
 }
 
-func selfDestructNewAccountCost(accountExists bool, balance tosca.Value) tosca.Gas {
-	if !accountExists && balance != (tosca.Value{}) {
+func selfDestructNewAccountCost(accountEmpty bool, balance tosca.Value) tosca.Gas {
```

Comment:
Nit: now that we are cleaning this up. This function is just an if. We either inline it or we move the new balance&code&nonce inside.

Reply:
I think someone pulled it out for testing but I inlined it again and updated the tests.